reya_actions/actions/unstake.py

```diff
@@ -0,0 +1,34 @@
+from web3 import Web3
+from hexbytes import HexBytes
+from dataclasses import dataclass
+
+@dataclass
+class UnstakingParams:
+    shares_amount: int
+    min_tokens: int
+
+def unstake(config: dict, params: UnstakingParams):
+    w3 = config['w3']
+    account = config['w3account']
+    passive_pool = config['w3contracts']['passive_pool']
+    
+    # Unstake rUSD from the passive pool
+    tx_hash = passive_pool.functions.removeLiquidity(1, params.shares_amount, params.min_tokens).transact({'from': account.address})
+    tx_receipt = w3.eth.wait_for_transaction_receipt(tx_hash)
+    print(f'Unstaked from passive pool: {tx_receipt.transactionHash.hex()}')
+
+    # Decode the logs to get the resulting shares amount
```

Comment:
can't we parse the returned value of the tx?

Reply:
Receipt doesn't have the return data.